hazelcast/test/src/HazelcastTests2.cpp

```diff
@@ -468,15 +468,13 @@ namespace hazelcast {
 
                 auto h = [&called](const std::string &,
                                    const std::string &,
-                                   const char*,
-                                   int,
                                    logger::level,
                                    const std::string &) {
                     called = true;
                 };
 
                 logger_config.handler(h);
-                logger_config.handler()("", "", "", 0, logger::level::fine, "");
+                logger_config.handler()("", 0, logger::level::fine, "");
```

Comment:
```suggestion
                logger_config.handler()("", "", logger::level::fine, "");
```


Can you also change the name of this test? I realized I made a typo there, it should have been `testSetGetLogHandler` but I somehow wrote `testSetGetLoadHandler`.